package.json

```diff
@@ -92,6 +92,18 @@
     "wrap-ansi": "^9.0.0",
     "ws": "^8.14.2"
   },
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "peerDependenciesMeta": {
+    "react": {
+      "optional": true
+    },
+    "react-dom": {
+      "optional": true
+    }
+  },
```

Comment:
I don’t think this is needed now that we use react from `npm:react` instead of node_modules. 🤔